### DISCUSSION TITLE
Is there a way to limit output size? I am running into some issues with massive output

### DISCUSSION THREAD

**Discussion by S4lt5** [projectdiscovery/nuclei#4589] [Q&A] [answered]
Hi. I'm rather happily using nuclei with my fork of nuclear pond and control-rod for the frontend and doing scanning in AWS using Lambda.

One issue I seem to be having trouble with, is really massive return values when using:

```
-jsonl -o foo.json
```
I will often get really massive output results, which seem to capture the entire main page.

Is there any way in nuclei to take e.g. the first XYZ Bytes or something from a template hit? Some new templates seem to just capture the entire website as output such as 

https://github.com/projectdiscovery/nuclei-templates/blob/fb4ccaa321be4ef29a9c5c4a3aaa62f7d1dee75e/http/misconfiguration/mixed-active-content.yaml

I think I am running into some sort of maximum output length from my lambda but at any rate it sorta breaks everything for me and generally speaking I never really want a ~100k capture in my output.

If this doesn't exist in some factor, I guess I can try to patch it in on my end, but figured I'd start the conversation by asking.

**Comment by ehsandeep** [comment, marked as answer]
@S4lt5 

```
   -rsr, -response-size-read int         max response size to read in bytes (default 10485760)
   -rss, -response-size-save int         max response size to read in bytes (default 1048576)
   ```
